scripts/lint_gal.py

```
@@ -0,0 +1,197 @@
+    """Return normalized review_v1 findings plus non-normative metrics."""
+    findings: list[dict] = []
+    prose = prose_text(text)
+
+    for match in SERVICE_NOMINALIZATION.finditer(prose):
+        findings.append(_finding(
+            rule_id="GAL-KANZ-VERB",
+            phenomenon_id="editing.action_hidden_in_nominalization",
+            excerpt=match.group(0),
+            line=_line(prose, match.start()),
+            reason=("По системе Норы Галь это кандидат на проверку: действие может быть спрятано "
+                    "в служебном глаголе и отглагольном существительном. Не менять механически "
```

Ruff (0.16.1)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)